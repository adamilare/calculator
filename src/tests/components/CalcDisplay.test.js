import React from 'react';
import renderer from 'react-test-renderer';
import CalcDisplay from '../../components/calcDisplay'; // eslint-disable-line

test("Calculator's output component displays correctly", () => {
  const tree = renderer.create(<CalcDisplay />).toJSON();
  expect(tree).toMatchSnapshot();
});
