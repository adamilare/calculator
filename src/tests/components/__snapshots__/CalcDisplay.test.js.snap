// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator's output component displays correctly 1`] = `
<span
  className="calc-btn calc-display"
  data-testid="calc-display"
>
  0
</span>
`;
